code/modules/projectiles/guns/energy/special.dm

```diff
@@ -241,9 +249,16 @@
 			if(istype(WH))
 				WH.gun = WEAKREF(src)
 
-/obj/item/gun/energy/wormhole_projector/process_chamber()
-	..()
-	select_fire()
+/obj/item/gun/energy/wormhole_projector/afterattack(atom/target, mob/living/user, flag, params)
+	var/list/modifiers = params2list(params)
+	if(!LAZYACCESS(modifiers, RIGHT_CLICK) && select == AMMO_SELECT_ORANGE) //left click to fire blue wormholes.
```

Comment:
`RIGHT_CLICK` should never be being run through `afterattack`. It's only doing it because...